_docs/resources/sketch-library.md

```diff
@@ -0,0 +1,276 @@
+---
+
+## Sketch Library
+
+<section class="static-section" markdown="1">
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+There are two approaches to installing the Helix library:
+
+1. **You can install the Sketch library from your Google Drive.** The primary
+benefit of this approach is that you automatically receive updates made to the
+library.
+
+2. **You can install the Sketch library from your local machine.** This approach
+involves downloading a copy of the library from Github, saving it to
+your local machine, and then loading the library into Sketch. The primary
+benefit of this approach is that it is easier to set up, but you do not
+automatically receive updates and must download the library each time there is
+an update.
+
+**Note:** The Helix Team sends an email each time the Helix library is updated.
+If you are not a part of the Helix email distribution list, you can request
+access by emailing helix@rackspace.com.
+
+{% endcolumn %}
+
+</div>
+</section>
+
+<section class="static-section" markdown="1">
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12 %}
+
+### Install the Sketch library from Google Drive
+Complete the following tasks to install the Sketch library from your Google
+Drive.
+
+Before you begin:
+
+- Ensure that you have a Google Drive account. If you have a Gmail account, you
+have access to a Google Drive. For more information, see [Google Drive](https://www.google.com/drive/)
+
+- Install a licensed copy of [Sketch](http://www.sketchapp.com). If you do not have a copy of Sketch, email helix@rackspace.com to request a license.
+
+{% endcolumn %}
+
+</div>
+</section>
+
+### Task 1: Move the shared Sketch library to your Google Drive account
+
+<section class="static-section" markdown="1">
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+After you sign up for a Google Drive account, if the library has not already
+been made available to you, send an email to the Helix Team requesting access
+to the Helix Sketch library. The Helix Team will provide a copy of the library
+to your **Shared with me** folder.
+
+Complete the following task to move the shared library to My Drive.
+
+<p><b>1.</b> Log in to Google Drive, and click <b>Shared Driv</b>.</p>
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+   {% figure [caption:""] [class:"image bg-light border"] %}
+   ![]({{site.url}}/assets/images/install-sketch-library/shared-drive.png)
+   {% endfigure %}
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+<p><b>2.</b> Right-click the helix library and select <b>Add to My Drive</b></p>
```

Comment:
s/`helix library`/`Helix library` or `Helix Library`

Reply:
Updated to Helix Library across all instances.